hibernate-core/src/test/java/org/hibernate/session/runtime/check/CheckAnnotationReadTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.session.runtime.check;
+
+import javax.persistence.Entity;
+
+import org.hibernate.testing.TestForIssue;
+import org.hibernate.testing.junit4.BaseCoreFunctionalTestCase;
+import org.junit.Test;
+
+/**
+ * @author Fabio Massimo Ercoli
+ */
+@TestForIssue(jiraKey = "HHH-13604")
```

Comment:
Not a big deal, but using space chars right after '(' and before ')' would be consistent with existing coding format